src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -61,6 +61,10 @@ public class GeneratePackageManagerJava : AndroidTask
 		[Required]
 		public bool EnablePreloadAssembliesDefault { get; set; }
 
+		// This property should be required but it will require modifying `monodroid` first
+		//[Required]
+		public string AndroidBinUtilsDirectory { get; set; }
```

Comment:
I don't see a reference to GeneratePackageManagerJava in monodroid?

Reply:
There are none, but I got errors in commercial tests when the property was required.  My guess is that, for some reason, `AndroidBinUtilsDirectory` was empty in some tests with the commercial bits present.

Reply:
Hmm interesting.

Reply:
@grendello can we try adding the required back so I can see what the error was? It might be a build order issue which we are currently unaware of?

Reply:
Sure, will push a commit in a moment

Reply:
Looks like it worked this time, odd.  Well, good :)